src/libraries/System.Private.CoreLib/src/System/Numerics/ITrigonometricFunctions.cs

```diff
@@ -62,6 +62,54 @@ public interface ITrigonometricFunctions<TSelf>
         /// <returns>The value of <paramref name="degrees" /> converted to radians.</returns>
         static virtual TSelf DegreesToRadians(TSelf degrees)
         {
+            // We don't want to simplify this to: degrees * (Pi / 180)
```

Comment:
Added a lengthy comment to this API here and pointed the other implementations to it to help avoid people trying to change the algorithm without considering the edge cases.